src/components/stats/List.tsx

```diff
@@ -0,0 +1,337 @@
+import { useEffect, useState } from 'react'
+import styled, { css } from 'styled-components'
+
+import InfiniteScroll from 'react-infinite-scroll-component'
+
+import { OrderDirection, User_OrderBy } from '@/graphql-schema'
+import ENSOrAddress from '@/src/components/aelin/ENSOrAddress'
+import { Investors as InvestorsIcon } from '@/src/components/assets/Investors'
+import { Sponsors as SponsorsIcon } from '@/src/components/assets/Sponsors'
+import { Vouchers as VouchersIcon } from '@/src/components/assets/Vouchers'
+import { genericSuspense } from '@/src/components/helpers/SafeSuspense'
+import {
+  ButtonCSS,
+  ButtonPrimaryLightSm,
+} from '@/src/components/pureStyledComponents/buttons/Button'
+import { BaseCard } from '@/src/components/pureStyledComponents/common/BaseCard'
+import {
+  Cell,
+  HideOnDesktop,
+  HideOnMobileCell,
+  LinkCell,
+  LoadingTableRow,
+  Row,
+  TableBody,
+  TableHead,
+} from '@/src/components/pureStyledComponents/common/Table'
+import { Search as BaseSearch } from '@/src/components/pureStyledComponents/form/Search'
+import { SortableTH } from '@/src/components/table/SortableTH'
+import { ChainsValues, getNetworkConfig } from '@/src/constants/chains'
+import useAelinUsers from '@/src/hooks/aelin/useAelinUsers'
+import { useEnsLookUpAddress } from '@/src/hooks/useEnsResolvers'
+
+const HeaderWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  row-gap: 20px;
+
+  @media (min-width: ${({ theme }) => theme.themeBreakPoints.tabletLandscapeStart}) {
+    flex-direction: row;
+  }
+`
+const SearchWrapper = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+`
+const ButtonsWrapper = styled.div`
+  display: flex;
+  flex: 1;
+  gap: 20px;
+`
+const Search = styled(BaseSearch)`
+  margin-bottom: 20px;
+  max-width: 100%;
+`
+
+const ButtonOnCSS = css`
+  background-color: ${({ theme }) => theme.buttonPrimary.backgroundColor};
+  border-color: ${({ theme }) => theme.buttonPrimary.borderColor};
+  color: ${({ theme }) => theme.buttonPrimary.color};
+
+  .fill {
+    fill: ${({ theme }) => theme.buttonPrimary.borderColor};
+  }
+`
+const TabButton = styled.div<{ active?: boolean }>`
+  ${ButtonCSS}
+  color: ${({ theme: { colors } }) => colors.textColor};
+  padding: 0 10px;
+  width: 110px;
+  &:hover {
+    ${ButtonOnCSS}
+  }
+  ${({ active }) => active && ButtonOnCSS}
+`
+
+enum Section {
+  SPONSORS = 'sponsors',
+  VOUCHERS = 'vouchers',
+  INVESTORS = 'investors',
+}
+
+enum SectionFilter {
+  SPONSORS = 'poolsSponsored_not',
+  VOUCHERS = 'poolsVouched_not',
+  INVESTORS = 'poolsInvested_not',
+}
+
+const VoucherLinkButton = ({ id }: { id: string }) => {
+  const router = useRouter()
+  const { data: voucherEnsAddress, isValidating } = useEnsLookUpAddress(id)
+
+  return (
+    <ButtonPrimaryLightSm
+      disabled={isValidating}
+      onClick={(e) => {
+        e.preventDefault()
+        e.stopPropagation()
+        router.push(`/?voucher=${voucherEnsAddress}`)
```

Comment:
I think we can use Shallow Routing here. Ref: https://nextjs.org/docs/routing/shallow-routing

Reply:
good call